### PR TITLE
Make animation color like mockup

### PR DESCRIPTION
**Pull request by aenario**
Before sending your pull-request 💌 make sure :

- [x] The tests are up-to-date and your code is tested

Screenshot_20200508-171720

### CODE REVIEW

File: src/Components/Animation.js [hunk 1]
```diff
@@ -332,7 +332,7 @@ class Animation extends React.Component {
 
     return (
       <View style={[styles.mainContainer, { height: this.animationHeight }, { width: this.animationWidth }]}>
-        {this._display(this.cuts)}
+        {!this.props.readonly && this._display(this.cuts)}
```
Comment (sallesma) [inline src/Components/Animation.js:335]: Not sure we should hide the cuts in the app. I think it helps understand the movements of the players

Comment (guillaumemoby) [reply src/Components/Animation.js:335]: As on ultiplays I am on the deleting the cut

Comment (sallesma) [reply src/Components/Animation.js:335]: Ok, not so important to me, let's go this way

Comment (sallesma) [reply src/Components/Animation.js:335]: (Just for the record and later discution, the cuts are displayed on ultiplay at the end)

Screenshot 2020-05-08 at 22 19 43

**Pull request merged**